workbench/bootstrap/providers.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+return [
+    \App\Providers\AppServiceProvider::class,
+    \Laravel\Horizon\HorizonServiceProvider::class,
```

Comment:
The horizon service provider is not auto-detected. When installing horizon, it adds the service provider to the providers file.